src/status_im2/contexts/communities/actions/accounts_selection/view.cljs

```diff
@@ -70,12 +72,20 @@
                                           :data accounts}
                       :description-props {:text (i18n/label :t/all-addresses)}}
                      {:title             (i18n/label :t/for-airdrops)
-                      :on-press          not-implemented/alert
+                      :on-press          #(rf/dispatch
+                                           [:show-bottom-sheet
+                                            {:content (fn [] [airdrop-addresses/view
+                                                              {:accounts       accounts
+                                                               :logo-uri       (get-in images
+                                                                                       [:thumbnail :uri])
+                                                               :community-name name}])}])
                       :description       :text
                       :action            :arrow
                       :label             :preview
                       :label-props       {:type :accounts
-                                          :data (take 1 accounts)}
+                                          ;; TODO: Substitute the provided data with the selected airdrop
```

Comment:
please create an issue and link in the TODO.  👍

Reply:
We have the issue already, gonna link it. Good idea!

Reply:
I recall there was a discussion about discontinuing the use of TODO comments and relying solely on GitHub issues to track tasks.

Reply:
Yeah it's preferable to do that 👌